ui/src/app/components/demographic-survey-v2.tsx

```diff
@@ -860,41 +871,41 @@ export const DemographicSurvey = fp.flow(withProfileErrorModal)(
             question='5. Are you deaf or do you have serious difficulty hearing?'
             selected={survey.disabilityHearing}
             onChange={(value) => onUpdate('disabilityHearing', value)}
-            style={{ marginBottom: '1rem' }}
+            style={{ marginBottom: '0.3rem' }}
```

Comment:
can make this a const style so changing it updates all of them

or we could go further and make this the default style for `YesNoOptionalQuestion` or `MultipleChoiceQuestion`